.github/workflows/e2e-nextjs.yml

```diff
@@ -0,0 +1,83 @@
+name: Next.js Integration test
+
+on:
+  pull_request:
+    branches: [main]
+    # Run on release PRs only
+    paths: ['CHANGELOG.md']
+
+jobs:
+  integration:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        bundler: ['default', 'esbuild']
+    steps:
+      - name: Checking out zip-it-and-ship-it
+        uses: actions/checkout@v2
+        with:
+          path: zip-it-and-ship-it
+      - name: Checking out netlify-plugin-nextjs
+        uses: actions/checkout@v2
+        with:
+          repository: netlify/netlify-plugin-nextjs
+          path: netlify-plugin-nextjs
+      - name: Checking out netlify-cli
+        uses: actions/checkout@v2
+        with:
+          repository: netlify/cli
+          path: netlify-cli
+      - name: Installing Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - name: Installing netlify-cli
+        run: npm ci
+        working-directory: netlify-cli
+      - name: Installing netlify-plugin-nextjs
+        run: npm install
+        working-directory: netlify-plugin-nextjs
+      - name: Installing netlify-plugin-nextjs demo
+        run: npm install
+        working-directory: netlify-plugin-nextjs/demo
+      - name: Patching netlify.toml to change the bundler
+        if: "${{ matrix.bundler != 'default' }}"
+        run:
```

Comment:
What do you think on extracting this into a dedicate JS file?

Reply:
I agree that it looks a bit ugly, but I do like the idea of having the entire flow contained in this file. I'm happy to revisit, though.

Reply:
I think we can revisit if the script gets more complicated